src/routes/OverviewPage/Header.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  Button,
+  ButtonVariant,
+  Divider,
+  Flex,
+  FlexItem,
+  Icon,
+  PageSection,
+  PageSectionVariants,
+  Text,
+  TextContent,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import service from '../../assets/ACS_service_icon.svg';
+
+function Header() {
+  return (
+    <PageSection variant={PageSectionVariants.light}>
+      <Flex>
+        <FlexItem alignSelf={{ default: 'alignSelfCenter' }}>
+          <img src={service} alt="ACS service" />
```

Comment:
I think the alt text should either specify what the image is--the "ACS service logo"--or more correctly be an empty string, because it's redundant with the semantically correct header in the next section.
```suggestion
          <img src={service} alt="" />
```

reference: https://www.w3.org/WAI/tutorials/images/functional/#logo-image-within-link-text